src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  mwaNodeServer: 'http://localhost:1234'
```

Comment:
dont use static host

Reply:
this is configuration, not static.
tell me how to get backend server url in angular?

Reply:
done https://github.com/vnts7/mwa-prj/pull/6/commits/4eb9fdb6141c2aab63f56ec7a7df62b8a66f5ec9